public/manifest.json

```diff
@@ -39,5 +39,37 @@
         }
       ]
     }
-  }
+  },
+  "screenshots": [
+    {
+      "src": "/screenshots/screenshot-1.png",
```

Comment:
instead of naming the screenshots as `screenshot{index}.png` let's name it based on what in contains, like `virtual-whiteboard` , `wirframe`, `illustration-notes`,`export` etc.